_docs-v3/toolbar/titleRangeSeparator.md

```diff
@@ -0,0 +1,34 @@
+---
+title: titleRangeSeparator
+---
+
+The date range in the **agendaWeek** title is, by default, separated by "-" between the beginning and end date.
```

Comment:
the actual values you are describing are `" - "` and " to " . they contain a space on either side.